.github/workflows/publish_docs.yml

```diff
@@ -0,0 +1,100 @@
+name: Publish Docs to Github Pages
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  docs-agentkit:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./cdp-agentkit-core
+    strategy:
+      matrix:
+        python: ['3.10', '3.11', '3.12']
```

Comment:
we don't need to build the docs for multiple python versions. We can stick with `3.10`

Reply:
roger that